website/content/docs/upgrading/index.mdx

```diff
@@ -115,8 +129,11 @@ Upgrading installations of Vault which participate in [Enterprise Replication](/
 - When satisfied with functionality of upgraded secondary instances, upgrade
   the primary instance
 
-Upgrades of multiple replicated clusters can be complicated. Here is an
-example.
+~> **Note:** It is not safe to replicate from a newer version of Vault to an
+older version. When upgrading replicated clusters, ensure that upstream clusters
+are always on older versions of Vault than downstream clusters.
```

Comment:
```suggestion
this is a test
```

Reply:
Going to take "this is a test" at face value.